core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -119,11 +115,9 @@ public InodeTreePersistentState(InodeStore inodeStore, InodeLockManager lockMana
     mInodeStore = inodeStore;
     mInodeLockManager = lockManager;
     mTtlBuckets = ttlBucketList;
-    mCreatedFileHistogram = null;
-    mRemovedFileHistogram = null;
-    mFileSizeHistogram = new Histogram(2);
-    mRemovedFileRecorder = new Recorder(2);
-    mCreatedFileRecorder = new Recorder(2);
+    mBucketCounter = new BucketCounter(
+        ImmutableList.<Long>builder().add(1024L, 1024L * 1024L, 1024L * 1024L * 1024L,
```

Comment:
Can you make this configurable?
Leveraging FormatUtils.parseSpaceSize(), we could allow inputs like: "1KB, 1MB, 10MB, 100MB, 1GB, 10GB, 100GB".

Reply:
done